### PR TITLE
[#1789] fix(mysql): The table properties values extracted from MySQL need to go through the Gravitino conversion logic.

### PR DESCRIPTION
**Pull request by Clearvive**
### What changes were proposed in this pull request?

The table properties values extracted from MySQL need to go through the Gravitino conversion logic.

### Why are the changes needed?
Fix: #1789 

### Does this PR introduce _any_ user-facing change?
NO
### How was this patch tested?
Conducted end-to-end testing

### CODE REVIEW

**Review by yuqi1129 — COMMENTED** [review]
LGTM

File: catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/JdbcCatalogOperations.java [hunk 1]
```diff
@@ -268,25 +268,26 @@ public Table loadTable(NameIdentifier tableIdent) throws NoSuchTableException {
...
     String comment = load.comment();
     StringIdentifier id = StringIdentifier.fromComment(comment);
     if (id == null) {
       LOG.warn(
           "The table {} comment {} does not contain gravitino id attribute", tableName, comment);
-      return load;
+    } else {
+      properties = StringIdentifier.newPropertiesWithId(id, properties);
+      // Remove id from comment
+      comment = StringIdentifier.removeIdFromComment(load.comment());
```
Comment (FANNG1) [inline catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/JdbcCatalogOperations.java:283]: load.comment() -> comment?

**Comment by FANNG1** [comment]
LGTM

**Review by yuqi1129 — APPROVED** [review]

**Comment by yuqi1129** [comment]
```shell
curl -X POST -H "Content-Type: application/json" -d '{"name":"t22","comment":"my test table","columns":[{"name":"integer1","type":"integer","comment":"id column comment","nullable":false, "autoIncrement": true},{"name":"long1","type":"long","comment":"name column comment","nullable":true},{"name":"float1","type":"float","comment":"name column comment","nullable":true},{"name":"double1","type":"double","comment":"name column comment","nullable":true},{"name":"decimal1","type":"decimal(20, 4)","comment":"name column comment","nullable":true},{"name":"date1","type":"date","comment":"name column comment","nullable":true},{"name":"time1","type":"time","comment":"name column comment","nullable":true},{"name":"string1","type":"string","comment":"name column comment","nullable":true},{"name":"binary1","type":"binary","comment":"name column comment","nullable":true}],"properties":{}, "indexes": [{"indexType": "PRIMARY_KEY", "name": "PRIMARY", "fieldNames":[["integer1"]]}]}' http://localhost:8090/api/metalakes/test/catalogs/mysql_catalog/schemas/db/tables
```

MySQL table created by Gravitino
```
CREATE TABLE `t22` (
  `integer1` int NOT NULL AUTO_INCREMENT COMMENT 'id column comment',
  `long1` bigint DEFAULT NULL COMMENT 'name column comment',
  `float1` float DEFAULT NULL COMMENT 'name column comment',
  `double1` double DEFAULT NULL COMMENT 'name column comment',
  `decimal1` decimal(20,4) DEFAULT NULL COMMENT 'name column comment',
  `date1` date DEFAULT NULL COMMENT 'name column comment',
  `time1` time DEFAULT NULL COMMENT 'name column comment',
  `string1` text COMMENT 'name column comment',
  `binary1` binary(1) DEFAULT NULL COMMENT 'name column comment',
  PRIMARY KEY (`integer1`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci COMMENT='my test table (From Gravitino, DO NOT EDIT: gravitino.v1.uid2141566772204103210)'
```

**Comment by yuqi1129** [comment]
My point is that we need to clearly explain this difference and make it clear to users. 
@jerryshao , what's your opinion?

**Pull request merged**